<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5" stretch="5,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3,2,9">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="workdayLabel">
              <property name="font">
               <font>
                <family>Comfortaa</family>
                <pointsize>30</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>Thur</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="dateLabel">
              <property name="font">
               <font>
                <family>Comfortaa</family>
                <pointsize>30</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>June 30</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="yearLabel">
              <property name="font">
               <font>
                <family>Comfortaa</family>
                <pointsize>30</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>2024</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="timeLabel">
              <property name="font">
               <font>
                <family>Comfortaa</family>
                <pointsize>50</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>21:09</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="todolistStrLabel">
                <property name="font">
                 <font>
                  <family>Comfortaa</family>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>todoList</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="4,8">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="Line" name="todoLine">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
</string>
                </property>
                <property name="lineWidth">
                 <number>3</number>
                </property>
                <property name="midLineWidth">
                 <number>3</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,4">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QTableWidget" name="todoTableWidget">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
                </property>
                <property name="autoScroll">
                 <bool>false</bool>
                </property>
                <property name="showGrid">
                 <bool>false</bool>
                </property>
                <property name="cornerButtonEnabled">
                 <bool>false</bool>
                </property>
                <attribute name="horizontalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderMinimumSectionSize">
                 <number>0</number>
                </attribute>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>100</number>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderMinimumSectionSize">
                 <number>0</number>
                </attribute>
                <attribute name="verticalHeaderDefaultSectionSize">
                 <number>22</number>
                </attribute>
                <row>
                 <property name="text">
                  <string>新建行</string>
                 </property>
                </row>
                <column>
                 <property name="text">
                  <string>新建列</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>新建列</string>
                 </property>
                </column>
                <item row="0" column="0">
                 <property name="text">
                  <string>fix the bug</string>
                 </property>
                </item>
                <item row="0" column="1">
                 <property name="text">
                  <string>Today</string>
                 </property>
                 <property name="textAlignment">
                  <set>AlignTrailing|AlignVCenter</set>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,5,3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="humidityLabel">
                <property name="font">
                 <font>
                  <family>Comfortaa</family>
                  <pointsize>30</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>23</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
              <item>
               <widget class="QLabel" name="weatherIconLabel">
                <property name="styleSheet">
                 <string notr="true">image: url(:/ui_src/icons/ui_src/icons/mti-大风.png);</string>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="tempLabel">
                <property name="font">
                 <font>
                  <family>Comfortaa</family>
                  <pointsize>50</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>23</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="futureTable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string/>
             </property>
             <property name="background">
              <color alpha="0">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
             <property name="background">
              <color alpha="0">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </property>
             <property name="foreground">
              <brush brushstyle="NoBrush">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
             <property name="background">
              <color alpha="0">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </property>
             <property name="foreground">
              <brush brushstyle="NoBrush">
               <color alpha="220">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
             <property name="background">
              <color>
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </property>
             <property name="foreground">
              <brush brushstyle="NoBrush">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </property>
            </row>
            <column>
             <property name="text">
              <string/>
             </property>
             <property name="background">
              <color>
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </property>
             <property name="foreground">
              <brush brushstyle="NoBrush">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </property>
            </column>
            <column>
             <property name="text">
              <string/>
             </property>
            </column>
            <column>
             <property name="text">
              <string/>
             </property>
            </column>
            <column>
             <property name="text">
              <string/>
             </property>
             <property name="foreground">
              <brush brushstyle="NoBrush">
               <color alpha="130">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>Thu</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="magicMirror_src.qrc">
               <normaloff>:/ui_src/icons/ui_src/icons/mti-大风.png</normaloff>:/ui_src/icons/ui_src/icons/mti-大风.png</iconset>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>27.2</string>
             </property>
            </item>
            <item row="0" column="3">
             <property name="text">
              <string>19.2</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>Fri</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="magicMirror_src.qrc">
               <normaloff>:/ui_src/icons/ui_src/icons/mti-大风.png</normaloff>:/ui_src/icons/ui_src/icons/mti-大风.png</iconset>
             </property>
            </item>
            <item row="1" column="2">
             <property name="text">
              <string>27.2</string>
             </property>
            </item>
            <item row="1" column="3">
             <property name="text">
              <string>19.2</string>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>Sat</string>
             </property>
             <property name="foreground">
              <brush brushstyle="NoBrush">
               <color alpha="180">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="magicMirror_src.qrc">
               <normaloff>:/ui_src/icons/ui_src/icons/mti-大风.png</normaloff>:/ui_src/icons/ui_src/icons/mti-大风.png</iconset>
             </property>
            </item>
            <item row="2" column="2">
             <property name="text">
              <string>27.2</string>
             </property>
            </item>
            <item row="2" column="3">
             <property name="text">
              <string>19.2</string>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string>Sun</string>
             </property>
             <property name="foreground">
              <brush brushstyle="NoBrush">
               <color alpha="130">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </property>
            </item>
            <item row="3" column="2">
             <property name="text">
              <string>27.2</string>
             </property>
            </item>
            <item row="3" column="3">
             <property name="text">
              <string>19.2</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="photoWidget" native="true">
            <widget class="QLabel" name="label_10">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>80</y>
               <width>66</width>
               <height>18</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="wordsLabel">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Talk is cheap. Show me the code.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="magicMirror_src.qrc"/>
 </resources>
 <connections/>
</ui>
